config/core/configmaps/default-pingsource.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: default-ping-webhook
```

Comment:
I'd rather pick a different name. Almost all CMs in evneting start with `config-` - except for `default-ch-webhook`, unfortunately

Reply:
@matzew Ok I followed that. Give the importance of this conformance can you open a issue for this?

Reply:
I've reconsidered. I think the confusion was I copied the example comment block. I'm now following what was done with default-broker.yaml,  default-broker-channel.yaml,  and default-channel.yaml